### PR TITLE
Enable the json resource to accept command output or JSON content

### PR DESCRIPTION
**Pull request by jerearista**

### CODE REVIEW

File: lib/resources/json.rb [hunk 1]
```diff
@@ -21,19 +30,28 @@ class JsonConfig < Inspec.resource(1)
 
     def initialize(path)
```
Comment (chris-rock) [inline lib/resources/json.rb]: should we rename `path` to `opts` then? Its only a path if used as a string then

**Comment by chris-rock** [comment]
Great improvement @jerearista

File: lib/resources/json.rb (hunk 1 above)
Comment (jerearista) [reply lib/resources/json.rb]: Agreed.   Changed.

**Comment by chris-rock** [comment]
Thanks @jerearista

**Pull request merged**